zanata-war/src/main/java/org/zanata/action/VersionHome.java

```diff
@@ -370,7 +370,6 @@ public String createVersion() {
     }
 
     public void copyVersion() {
-        getInstance().setStatus(EntityStatus.READONLY);
```

Comment:
The problem with that is when copy version completed, the UI still shows the version as "READONLY" due to the hibernate cache. 
Perhaps I should redirect to project page if user trying to access the version page? what do you think?